src/finiteVolume/processes/twoPhaseEulerAdvection.cpp

```diff
@@ -67,9 +232,16 @@ PetscErrorCode ablate::finiteVolume::processes::TwoPhaseEulerAdvection::Compress
     PetscFunctionBeginUser;
     auto twoPhaseEulerAdvection = (TwoPhaseEulerAdvection *)ctx;
     // Compute the norm of cell face
-    PetscReal norm[3];
+    PetscReal norm[3], newnormal[3];
```

Comment:
Is this needed?

Reply:
Good point, I don't think this changed solved anything, so I will get rid of this